crates/viewer/re_view_spatial/src/max_image_dimension_subscriber.rs

```diff
@@ -13,6 +14,11 @@ use re_types::{
 pub struct MaxDimensions {
     pub width: u32,
     pub height: u32,
+
+    pub has_image: bool,
+    pub has_encoded_image: bool,
+    pub has_depth_image: bool,
+    pub has_video: bool,
```

Comment:
segmentation image is conspicuously missing from this list.

Reply:
didn't need it later on (see how this ends up being used)
but I agree that on this thing alone this is just weeeeeird! Let's add it, don't think that hurts!